packages/network-transport-tcp/network-transport-tcp.cabal

```diff
@@ -105,3 +104,27 @@ Test-Suite TestQC
                   DeriveDataTypeable
                   MultiParamTypeClasses
   default-language: Haskell2010
+
+executable chat-server
```

Comment:
I imagine that these executables were used in a test previously. 
Would you mind writing an issue about writing a test using these new executables?

Reply:
Created https://github.com/haskell-distributed/distributed-process/issues/472 

I was using them as examples of how to use `network-transport-tcp`. 
In general it would be useful to have a corpus of working examples for `distributed-process` see https://github.com/haskell-distributed/distributed-process/issues/357

Reply:
Thanks!